ReactAndroid/src/main/java/com/facebook/react/uimanager/events/TouchEvent.java

```diff
@@ -67,30 +69,33 @@ private void init(
       int viewTag,
       TouchEventType touchEventType,
       MotionEvent motionEventToCopy,
+      long gestureStartTime,
       float viewX,
       float viewY,
       TouchEventCoalescingKeyHelper touchEventCoalescingKeyHelper) {
     super.init(viewTag);
 
+    Assertions.assertCondition(gestureStartTime != Long.MIN_VALUE,
```

Comment:
Do you mean the com.facebook.react.bridge.SoftAssertions? I could not find 	SoftAssertions in .common package.